exercises/flatten-array/example.js

```diff
@@ -0,0 +1,7 @@
+export default class Flattener {
+  flatten(arr) {
+    return arr
+      .reduce((acc, el) => Array.isArray(el) ? acc.concat(this.flatten(el)) : acc.concat(el), [])
```

Comment:
I would spread the expressions in the ternary operator on multiple lines to improve the readability of the solution:

```js
.reduce((acc, el) => {
  return Array.isArray(el)
    ? acc.concat(this.flatten(el))
    : acc.concat(el)
}, [])
```

Btw, that's a very elegant usage of recursion to solve this issue, Mat!